Content.Server/Toilet/ToiletSystem.cs

```diff
@@ -122,15 +124,30 @@ private void OnInteractHand(EntityUid uid, ToiletComponent component, InteractHa
                 }
             }
 
-            // just want to up/down seat?
-            // check that nobody seats on seat right now
             if (TryComp<StrapComponent>(uid, out var strap) && strap.BuckledEntities.Count != 0)
                 return;
 
-            ToggleToiletSeat(uid, component);
             args.Handled = true;
         }
 
+        private void OnToggleSeatVerb(EntityUid uid, ToiletComponent component, GetVerbsEvent<AlternativeVerb> args)
+        {
+            string alterToiletSeatText = component.IsSeatUp ? Loc.GetString("toilet-seat-close") : Loc.GetString("toilet-seat-open");
+
+            var verb = new AlternativeVerb()
+            {
+                Act = () =>
+                {
+                    if (TryComp<StrapComponent>(uid, out var strap) && strap.BuckledEntities.Count != 0)
+                        return;
+                    ToggleToiletSeat(uid, component);
```

Comment:
Should be a generic "CanToggle" method that's also used above.

Reply:
Implemented but now I am getting a test fail on character deletion test.

Reply:
Just merge master